cmd/restore-operator/README.md

```diff
@@ -0,0 +1,135 @@
+# etcd restore operator
+
+## Overview
+
+The etcd-restore-operator can restore an etcd cluster on Kubernetes from backup.
+
+The overall workflow is:
+- Create the etcd-restore-operator
+- Create an `EtcdRestore` Custom Resource which triggers a restore request that specifies:
+  - the etcd cluster spec
+  - how to access the backup
+- The etcd-restore-operator will restore a new cluster from the backup
+- The etcd-operator takes over the management of the restored cluster
+
+Note that currently the etcd-restore-operator only supports restoring from backups saved on S3.
+
+**Prerequisite**
+- Have an etcd backup saved on S3. See the [etcd-backup-operator README][backup-operator-README] as one way to save a backup to S3.
+
+>Note: This demo uses the `default` namespace.
+
+### Deploy the etcd-restore-operator
+
+Create a deployment of the etcd-restore-operator:
+
+```sh
+kubectl create -f example/etcd-restore-operator/deployment.yaml
+```
+
+This should create the following resources:
```

Comment:
I pefer not using the word like `will`, `should`, and etcd. it is simpler to state:
`This creates the following resources:`

I am fine with either way. just a suggestion.

Reply:
cc/ @hasbro17 https://developers.google.com/style/tense

Reply:
Yeah I agree. I've actually changed up most of the doc to only be actionable steps now.
Like "Do this, Verify that"

Reply:
i saw. the changes look great.